src/utils/responsive/useMedia.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback, useEffect, useState } from 'react'
```

Comment:
https://github.com/streamich/use-media

https://github.com/jepser/use-match-media

https://github.com/streamich/react-use/blob/master/docs/useMedia.md


Plenty of options ?

Reply:
I took the one from react-hooks, do you prefer to add a dependency ?

Reply:
yes, less maintenance for us :)